### ISSUE TITLE
Strikethrough merged pull requests

### ISSUE THREAD

**Issue by lloydwatkin** [isaacs/github#313]
When mentioning a pull request in an issue / comment / pull request if the pull request has been merged then strikethrough the link text, e.g. https://github.com/node-xmpp/node-xmpp-client/pull/64 should be struck-through.

**Comment by cirosantilli** [comment]
+1. Like Bugzilla.

**Comment by cirosantilli** [comment]
Would they also have strike through if they are closed without merging?

**Comment by TPS** [comment]
Maybe they should be double-struck-through or X-ed out?  Or different color?

This issue definitely should have <kbd>__Parity__</kbd> label!

**Comment by cirosantilli** [comment]
@TPS it seems that those graphical effects don't have good CSS support like strikethrough though: http://stackoverflow.com/questions/6395890/how-to-double-strikeout-a-text-in-html | http://stackoverflow.com/jobs/119644/software-engineers-inc-distributed-systems-api-improbable

**Comment by TPS** [comment]
@cirosantilli I was trying to differentiate between the different _kinds_ of closing, which Bugzilla itself doesn't actually do.  Plain strikethrough for everything seems fine, unless 1 wants to display a bit more info on _why_ the issue/PR was closed.  Even color isn't particularly good as an indicator, due to accessibility problems.

**Comment by TPS** [comment]
Of course, something like **~~#closed-accomplished-bug#~~** vs ~~*#closed-invalid/wontfix-bug#*~~ should be ok, too.…

Is any1 else missing much typically-bold formatting all over GitHub.com today?  Weird!

**Comment by bfred-it** [comment]
[GitHub Issue Link Status](https://github.com/bfred-it/github-issue-link-status) will do that: 

<a href="https://github.com/bfred-it/github-issue-link-status/"><img src="https://github.com/bfred-it/github-issue-link-status/raw/master/screenshot.png" width="640"></a>
